server/config/config.js

```diff
@@ -4,7 +4,8 @@ const dbConfig = {
   database: process.env.POSTGRES_DB,
   host: process.env.POSTGRES_HOST,
   port: process.env.POSTGRES_PORT,
-  dialect: 'postgres'
+  dialect: 'postgres',
+  logging: process.env.SEQUELIZE_LOGGING === 'true' ? msg => console.log(msg) : false

```

Comment:
It would be cleaner to just pass `console.log`, rather than an inline function to call `console.log`:
```
logging: process.env.SEQUELIZE_LOGGING === 'true' ? console.log : false
```

Reply:
Indeed it would, I didn't think of that.